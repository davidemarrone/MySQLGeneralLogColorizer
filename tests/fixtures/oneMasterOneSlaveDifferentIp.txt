		  102 Connect	genericUser@127.0.0.1 on myDb
		  102 Query	SET NAMES UTF8, time_zone = "+0:00"
		  102 Query	SELECT * FROM customer WHERE idCustomer = 1
		  103 Connect	genericUser@192.168.1.1 on myDb
		  103 Query	SET NAMES UTF8, time_zone = "+0:00"
		  103 Query	SELECT * FROM customer WHERE username LIKE 'a%'
		  103 Query	SELECT email
                    FROM customer
                    WHERE idCustomer = 1
