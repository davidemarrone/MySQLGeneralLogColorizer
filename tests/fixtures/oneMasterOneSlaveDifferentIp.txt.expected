[0;94m		  102 Connect	genericUser@127.0.0.1 on myDb
[0m[0;31m		  102 Query	SET NAMES UTF8, time_zone = "+0:00"
[0m[0;31m		  102 Query	SELECT * FROM customer WHERE idCustomer = 1
[0m[0;94m		  103 Connect	genericUser@192.168.1.1 on myDb
[0m[0;32m		  103 Query	SET NAMES UTF8, time_zone = "+0:00"
[0m[0;32m		  103 Query	SELECT * FROM customer WHERE username LIKE 'a%'
[0m[0;32m		  103 Query	SELECT email
[0m[0;32m                    FROM customer
[0m[0;32m                    WHERE idCustomer = 1
[0m